python/src/NAIS_doc.i.in

```diff
@@ -0,0 +1,126 @@
+%feature("docstring") OT::NAIS
+
+"Non Adaptive Importance Sampling (NAIS) algorithm.
+
+Parameters
+----------
+event : :class:`~openturns.RandomVector`
+    Event we are computing the probability of failure.
```

Comment:
```suggestion
    Event we are computing the probability of.
```
Maybe "Failure event we are computing the probability of", but I would rather keep this generic.